mysite/ct/views.py

```diff
@@ -329,7 +329,14 @@ def course_view(request, course_id):
     'show courselets in a course'
     course = get_object_or_404(Course, pk=course_id)
     if is_teacher_url(request.path):
-        notInstructor = check_instructor_auth(course, request)
+        try:
```

Comment:
@alexbojko Again. This is the same repetition. I want to suggest you change `check_instructor_auth` to simply return Boolean value. For this please review all places where this function is used. Thanks.